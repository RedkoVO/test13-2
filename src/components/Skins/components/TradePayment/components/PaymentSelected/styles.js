import CancelІSelected from '../../../../../../assets/images/svg/cancelІSelected.svg'

export default () => ({
  root: {
    width: '240px',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'flex-end',
    paddingLeft: '30px',
    paddingRight: '20px',
    boxSizing: 'border-box'
  },
  paymentSelectedTitle: {
    fontSize: '12px',
    fontWeight: 500,
    alignSelf: 'flex-start',
    marginBottom: '7px'
  },
  paymentSelectedItem: {
    display: 'flex',
    width: '160px',
    paddingLeft: '5px',
    marginBottom: '2px',
    fontSize: '10px',
    fontWeight: 500,
    height: '11px',
    backgroundColor: 'rgba(0,0,0,0.13)',
    borderRadius: '5px'
  },
  selectedPrice: {
    fontSize: '10px',
    fontWeight: 500
  },
  selectedClose: {
    width: '8px',
    height: '8px',
    marginTop: '1px',
    marginLeft: '5px',
    background: `url(${CancelІSelected})`,
    backgroundRepeat: 'no-repeat',
    backgroundPosition: 'center',
    backgroundSize: 'contain'
  },
})
