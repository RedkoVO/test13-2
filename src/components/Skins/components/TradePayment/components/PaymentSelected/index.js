import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const PaymentSelected = ({ classes, selectedItems }) => (
  <div className={classes.root}>
    <div className={classes.paymentSelectedTitle}>You selected 6 items:</div>
    {selectedItems.map((item, index) => (
      <div className={classes.paymentSelectedItem} key={item.id}>
        <div className={classes.selectedText}>
          {index}) {item.title} ....
        </div>
        <div className={classes.selectedPrice}>{item.cost}</div>
        <div className={classes.selectedClose} />
      </div>
    ))}
  </div>
)

PaymentSelected.propTypes = {
  classes: PropTypes.object,
  selectedItems: PropTypes.array
}

export default withStyles(styles)(PaymentSelected)
