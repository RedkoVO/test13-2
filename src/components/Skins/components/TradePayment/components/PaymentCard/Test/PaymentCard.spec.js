import * as React from 'react'
import renderer from 'react-test-renderer'

import PaymentSelected from '../'

it('renders correctly', () => {
  const tree = renderer
    .create(<PaymentSelected />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
