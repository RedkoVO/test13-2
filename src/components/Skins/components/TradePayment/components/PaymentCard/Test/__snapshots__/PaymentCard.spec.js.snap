// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="PaymentCard-root-1"
>
  <div
    className="PaymentCard-paymentCardTitle-2"
  >
    Cashout on my Credit card:
  </div>
  <div
    className="PaymentCard-paymentInputWrap-3"
  >
    <input
      className="PaymentCard-paymentCardNum-5"
      placeholder="Card Number"
      type="text"
    />
  </div>
  <div
    className="PaymentCard-paymentInputWrap2-4"
  >
    <input
      className="PaymentCard-paymentCardHolder-6"
      placeholder="Card Holder"
      type="text"
    />
  </div>
  <div
    className="PaymentCard-paymentCardMonth-7"
  >
    MM
  </div>
  <div
    className="PaymentCard-paymentCardYear-8"
  >
    YYYY
  </div>
  <div
    className="PaymentCard-paymentInputWrap3-9"
  >
    <input
      className="PaymentCard-paymentCardCvc-10"
      placeholder="CVC"
      type="text"
    />
  </div>
</div>
`;
